atd-vze/src/views/Crashes/CrashCollapses.js

```diff
@@ -74,28 +74,21 @@ class CrashCollapses extends Component {
         <Row>
           <Col>
             <Card>
-              <CardHeader>
-                <i className="fa fa-align-justify" /> Related Records
-              </CardHeader>
-              <CardBody>
-                <div id="accordion">
-                  <UnitDetailsCard
-                    isExpanded={this.state.accordion[0]}
-                    toggleAccordion={this.toggleAccordion}
-                    {...this.props.props}
-                  />
-                  <PeopleDetailsCard
+              <UnitDetailsCard
+                isExpanded={this.state.accordion[0]}
+                toggleAccordion={this.toggleAccordion}
+                {...this.props.props}
```

Comment:
i started to wonder about the usage of `props.props` but that looks like an issue for another day 👍

Reply:
prop props 😄 - agreed!